### PR TITLE
Hide items in plant pots and plushies

### PR DESCRIPTION
**Pull request by GeneriedJenelle**
Also adds a thread and needle item. It can be useful, for future applications I suppose. On world we definitely have plans for it.

* You can see if an item is in a plant pot if you are next to it and examine it.
* You can hide tiny items in small plushies, and larger items in big plushies.
* Use a scalpel or any sharp knife to cut open a plushie, put an item inside (like candy, not bombs or anything) and sew it back up. 
* Small plushies and plant pots can only hold tiny items. Biggie plushies can hold bigger items.
* Use a needle and thread (buyable from gift vendor) to sew the plushies up when you're done. 
* For all your smuggling needs.
* Never trust plushies again.

image
image
image
image
image

### CODE REVIEW

**Comment by GeneriedJenelle** [comment]
Code should be good, just gotta upload the sewing thread icon when I get home.

**Comment by TheFurryFeline** [comment]
:eyes: Items in plushies? BOI! This'll be quite an interesting thing for those who like drug dealing RP stuff.

**Comment by GeneriedJenelle** [comment]
Icon added. All done.

**Comment by Mechoid** [comment]
Not having it for plushies makes sense, though I can see a purpose for having a borg-interaction be added for dumping / adding things from / to plant-pots via standard / gripper interaction respectively.

File: code/game/objects/structures/flora.dm [hunk 1]
```diff
@@ -222,6 +222,44 @@
...
+	..()
+	if(in_range(user, src) && stored_item)
+		to_chat(user, "<i>You can see something in there...</i>")
+
+/obj/structure/flora/pottedplant/attackby(obj/item/I, mob/user)
+	if(stored_item)
+		to_chat(user, "<span class='notice'>[I] won't fit in. There already appears to be something in here...</span>")
+		return
+
+	if(I.w_class > ITEMSIZE_SMALL)
```
Comment (Atermonera) [inline code/game/objects/structures/flora.dm]: > ITEMSIZE_SMALL allows small items into the pots

Comment (GeneriedJenelle) [reply code/game/objects/structures/flora.dm]: Would you prefer if it was ITEMSIZE_TINY for balancing?

Comment (Atermonera) [reply code/game/objects/structures/flora.dm]: You noted they could only hold tiny items in the PR desc, was merely pointing out an inconsistency. I don't have any qualms either way.

Comment (GeneriedJenelle) [reply code/game/objects/structures/flora.dm]: Yeah I amended it, let's see how it goes. I hope tape recorders can fit.

**Pull request merged**